packages/closer/components/BookingRequestButtons/index.tsx

```
@@ -38,62 +38,61 @@ const BookingRequestButtons = ({
             </Link>
           )}
-
         </>
       )}
-      {isSpaceHost && !Boolean(user && isBookingCancelable && user._id === createdBy) && (
-        <Link passHref href={`/bookings/${_id}/cancel`}>
-          <Button type="secondary">⭕ {t('booking_cancel_button')}</Button>
-        </Link>
-      )}
+      {isSpaceHost &&
+        !Boolean(user && isBookingCancelable && user._id === createdBy) && (
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)